data/src/main/java/co/nimblehq/compose/crypto/data/model/response/TrendingItemResponse.kt

```diff
@@ -0,0 +1,110 @@
+package co.nimblehq.compose.crypto.data.model.response
+
+import co.nimblehq.compose.crypto.domain.model.TrendingItem
+import com.squareup.moshi.Json
+
+data class TrendingItemResponse(
```

Comment:
🤔 Would it be possible to reuse `CoinItemResponse`? The UI mostly displays the same information.

Reply:
Yes, I agreed, @hoangnguyen92dn but There're some fields we don't use the same type e.g. `currentPrice`:

- In `CoinItemResponse` is `Int`.
- In `TrendingItemResponse` is `Double`.
- In  `CoinItemResponse` from this [PR](https://github.com/nimblehq/jetpack-compose-crypto/pull/27) is `BigDecimal`.

I think it's good to use the `BigDecimal` What do you think? 🙏  cc @luongvo

Reply:
I think we should use BigDecimal for all pricing fields, especially with coin pricing because we don't know exactly how big or small it is. @hoangnguyen92dn @Wadeewee

Reply:
It could a bit hard for you @Wadeewee to extend the modified CoinItem from my PR. How about 
- reviewing and merging my one #27 first 
- @Wadeewee can rebase this PR on my one #27 

@hoangnguyen92dn @Wadeewee? 🙏

Reply:
Yep, @luongvo Sound like a plan 🚀

Reply:
@luongvo @Wadeewee Agree that we should update to use `BigDecimal` for the coin price.